apps/dashboard/src/components/template-store/templates/authentication/email-verification-code.ts

```diff
@@ -23,6 +23,6 @@ export const emailVerificationCodeTemplate: WorkflowTemplate = {
     ],
     tags: [],
     active: true,
-    __source: WorkflowCreationSourceEnum.TEMPLATE_STORE,
+    source: WorkflowCreationSourceEnum.TemplateStore,
```

Comment:
why did we change `__source` to `source`?

Reply:
I think that speakeasy is doing this transformation, this also includes transforming "_id" to "id", basically everything with an _